client/src/frontend/components/resource-collection/Description/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FormattedDate from "global/components/FormattedDate";
+import * as Styled from "./styles";
+
+export default function Description({ date, description }) {
+  const { t } = useTranslation();
+
+  if (!date && !description) return null;
+
+  return (
+    <Styled.Wrapper>
+      {date && (
+        <Styled.DateWrapper>
+          <FormattedDate
+            prefix={t("dates.collection_created")}
+            format="MMMM yyyy"
+            date={date}
+          />
+        </Styled.DateWrapper>
+      )}
+      {description && (
+        <Styled.Description
+          dangerouslySetInnerHTML={{
+            __html: description
+          }}
+        />
+      )}
```

Comment:
This rendered for me when `description` was just an empty string. Might need to check `!!description` instead.